sim/druid/item_sets_pve_phase_4.go

```diff
@@ -236,17 +238,7 @@ var ItemSetCenarionRage = core.NewItemSet(core.ItemSet{
 
 // Bear Form and Dire Bear Form increase all threat you generate by an additional 20%, and Cower now removes all your threat against the target but has a 20 sec longer cooldown.
 func (druid *Druid) applyT1Guardian6PBonus() {
-	label := "S03 - Item - T1 - Druid - Guardian 6P Bonus"
-	if druid.HasAura(label) {
-		return
-	}
-
-	druid.RegisterAura(core.Aura{
-		Label: label,
-		OnInit: func(aura *core.Aura, sim *core.Simulation) {
-			// TODO
-		},
-	})
+	druid.CenarionRageThreatBonus = .2
 }
```

Comment:
This will double dip without the aura code here. What you can do for this bonus is in the `OnInit`, grab the old `OnGain` and `OnExpire` for bear form, then override them, calling the old ones and then also adding the threat mod here:
```
OnInut: func(aura *core.Aura, sim *core.Simulation) {
  oldOnGain := druid.BearFormAura.OnGain
  druid.BearFormAura.OnGain = func(aura *core.Aura, sim *core.Simulation) {
    oldOnGain(aura, sim)
    druid.Pseudostats.ThreatMultiplier += 0.20
  }

 oldOnExpire := druid.BearFormAura.OnExpire
 druid.BearFormAura.OnExpire = func(aura *core.Aura, sim *core.Simulation) {
   oldOnExpire(aura, sim)
   druid.Pseudostats.ThreatMultiplier -= 0.20
 }
}
```